docs/integrations.md

```diff
@@ -0,0 +1,9 @@
+# Integrations 
+
+## Geras (OpenTSDB integration)
```

Comment:
Awesome. I would split this (even though it's one) in the sections. So let's add `## StoreAPI` then let's add something like

```
[StoreAPI](link to proto) is a common proto interface for gRPC component that can connect to [Querier](link to components/querier) in order to fetch the metric series. Natively Thanos implements [Sidecar](link to components/sidecar) (local Prometheus data), [Ruler](link to componenets/rule) and [Store gateway](link to components/store). This solves fetching series from Prometheus or Prometheus TSDB format, however same interface can be used to fetch metrics from other storages. 

Below you can find some public integrations with other systems through StoreAPI:

### OpenTSDB

(geras...) 

```
Can you be clear if it supports downsampled (aggregated chunks) or is it planned. It is totally fine if not (:

Reply:
do you mean down sampled queries? Geras basically just translates the PromQL query to OpenTSDB query and passes the data. it's possible to aggregate the data on OpenTSDB level, but it's not supported at the moment.

Reply:
mentioned it in the doc: 
| Although OpenTSDB is able to aggregte the data, it's not supported by Geras at the moment.